package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-chat-window",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
whoops....

Reply:
But https://www.npmjs.com/package/react-chat-window says 1.2.1 as of one month ago...I don't understand...